s3-sqs-lambda-terraform/example-pattern.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "S3 to Step Functions using EventBridge",
+  "description": "How to create an EventBridge rule with S3 as the event source and Step Functions as target.",
+  "language": "nodejs",
+  "level": "200",
+  "framework": "Terraform",
+  "introBox": {
+    "headline": "How to create an EventBridge rule with S3 as the event source and Step Functions as target.",
```

Comment:
Please update to match the actual pattern

Reply:
Sorry bout the confusion, updated.

Reply:
This still seems to be unresolved